compiler/src/dotty/tools/backend/jvm/DottyBackendInterface.scala

```diff
@@ -224,7 +224,8 @@ class DottyBackendInterface(outputDirectory: AbstractFile, val superCallsMap: Ma
   def shouldEmitAnnotation(annot: Annotation): Boolean = {
     annot.symbol.isJavaDefined &&
       retentionPolicyOf(annot) != AnnotationRetentionSourceAttr &&
-      annot.args.isEmpty
+      annot.args.isEmpty &&
+      !annot.tree.isInstanceOf[Block] // unblocking Olaf, needs a proper fix
```

Comment:
Does this mean the `@Fork` annotation is not emitted to bytecode? I think the annotation is necessary for the jmh benchmarks to run.

It might also be good to reference the issue in the comment.